lib/bindings/HTTPBindings.js

```diff
@@ -406,11 +408,70 @@ function start(callback) {
         router: express.Router()
     };
 
+    var options = {
+        swaggerDefinition: {
+            info: {
+                title: 'IoT Agent UL2 - HTTP', // Title (required)
+                version: '1.0.0', // Version (required)
```

Comment:
This refers to the version of the IOTA? In that case, it should be obtained from version field in package.json.

Reply:
It is related to Swagger documentation version.

Reply:
I think it would be a good idea to tie the version of the swagger documentation to the version of the IOTA Agent. It would be simpler.

The only case in which makes sense to have a version for the documentation and a version for the IOTA is the one in which the same IOTA version (from a code point of view) could have different documentation versions, which I think is not realistic.

Reply:
The version of the swagger documentation has been changed to the same IOTA UL version how as requested but is important to know that this version really is the first documentation version.

Reply:
fixed in 2164dc01ef8494fe5ab99c5da6956cd8fa9b9094